examples/healthData/package.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "healthdata",
```

I can't quite imagine where this is going to be a problem, but the fact that the containing folder has an a name that is distinct by casing on Unix, but not on Windows, might be a problem long term. Let's just call it `health-data` in the meantime?

```suggestion
  "name": "health-data",
```

then re-run `npm install` from the root and update the `package-lock.json`